tsl/test/sql/compression_errors.sql

```diff
@@ -99,6 +99,7 @@ ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_segmentby = 'ran
 ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_segmentby = 'c LIMIT 1');
 ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_segmentby = 'c + b');
 ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_orderby = 'a, p');
+ALTER TABLE foo set (timescaledb.compress, timescaledb.compress_segmentby = 'b, b');
```

Comment:
Could you add a test if duplicated `compress_orderby` have the same problem?

Reply:
Thanks @mkindahl for pointing this. I have updated the patch to handle duplicate columns as part of both `compress_segmentby` and `compress_orderby` options.